pkgs/top-level/all-packages.nix

```diff
@@ -27286,4 +27286,6 @@ in
   gpio-utils = callPackage ../os-specific/linux/kernel/gpio-utils.nix { };
 
   navidrome = callPackage ../servers/misc/navidrome {};
+
+  urserver = callPackage ../servers/urserver { };
```

Comment:
please attempt to sort this, it causes merge conflicts if everyone appended to the end.

Reply:
Moved this line to the 'servers' section at the right position, and split up the commit.